### ISSUE TITLE
Color contrast audit

### ISSUE THREAD

**Issue by karatechops** [hpe-design/design-system#251]
![Screen Shot 2020-01-17 at 5 18 54 PM](https://user-images.githubusercontent.com/5983843/72650116-7a93e800-394d-11ea-824f-d7ba140c3026.png)

![Screen Shot 2020-01-17 at 5 19 40 PM](https://user-images.githubusercontent.com/5983843/72650153-9a2b1080-394d-11ea-97b8-83622828cda7.png)

![Screen Shot 2020-01-17 at 5 19 27 PM](https://user-images.githubusercontent.com/5983843/72650154-9a2b1080-394d-11ea-9aa3-08ea1a37fc8e.png)

![Screen Shot 2020-01-17 at 5 19 18 PM](https://user-images.githubusercontent.com/5983843/72650155-9ac3a700-394d-11ea-9d4e-8fa436d768d5.png)

**Comment by taysea** [comment]
Also related: #200

**Comment by taysea** [comment]
I did an audit on all of the pages in both dark and light mode for color contrast. These are the places where there was an explicit error for color contrast. However, in addition to these I think we should address the above noted areas the subjectively lack contrast (Such as the search placeholder text).

**Dark Mode**

- Landing page: Descriptive text under develop
<img width="363" alt="Screen Shot 2020-01-23 at 3 18 42 PM" src="https://user-images.githubusercontent.com/12522275/73035555-5b89d000-3dfd-11ea-9e45-f9d5d4678a8d.png">

- Layout page under Components: Stack example
<img width="109" alt="Screen Shot 2020-01-23 at 4 28 51 PM" src="https://user-images.githubusercontent.com/12522275/73035586-78be9e80-3dfd-11ea-97e7-1e030378b1a8.png">

- Develop: HPE Design system text
<img width="273" alt="Screen Shot 2020-01-23 at 4 29 41 PM" src="https://user-images.githubusercontent.com/12522275/73035604-95f36d00-3dfd-11ea-8dc5-987fb2bf24db.png">

**Light Mode**

- Landing: Descriptive text under Foundation and Design
<img width="373" alt="Screen Shot 2020-01-23 at 4 34 27 PM" src="https://user-images.githubusercontent.com/12522275/73035765-43668080-3dfe-11ea-89ec-df40e7f56b51.png">
<img width="369" alt="Screen Shot 2020-01-23 at 4 34 30 PM" src="https://user-images.githubusercontent.com/12522275/73035766-43ff1700-3dfe-11ea-9fb4-c2b88089e40a.png">

- Foundation: HPE Design System
<img width="269" alt="Screen Shot 2020-01-23 at 4 35 23 PM" src="https://user-images.githubusercontent.com/12522275/73035817-642ed600-3dfe-11ea-94a9-112742196690.png">

- Design: HPE Design System
<img width="249" alt="Screen Shot 2020-01-23 at 4 35 17 PM" src="https://user-images.githubusercontent.com/12522275/73035830-6bee7a80-3dfe-11ea-9874-9d80f931620f.png">

- CallToAction Anchors: 
<img width="172" alt="Screen Shot 2020-01-23 at 4 37 25 PM" src="https://user-images.githubusercontent.com/12522275/73035906-a9eb9e80-3dfe-11ea-8d55-2293b858c10b.png">

**In general**
- Color Rows:
<img width="641" alt="Screen Shot 2020-01-14 at 12 39 57 PM" src="https://user-images.githubusercontent.com/12522275/73035979-da333d00-3dfe-11ea-8677-89a7dd72789d.png">
<img width="644" alt="Screen Shot 2020-01-14 at 12 40 21 PM" src="https://user-images.githubusercontent.com/12522275/73035980-da333d00-3dfe-11ea-8cf5-46cbe3d7b4cf.png">
<img width="642" alt="Screen Shot 2020-01-14 at 12 40 27 PM" src="https://user-images.githubusercontent.com/12522275/73035981-da333d00-3dfe-11ea-892d-f270994b628b.png">
<img width="646" alt="Screen Shot 2020-01-14 at 12 40 38 PM" src="https://user-images.githubusercontent.com/12522275/73035982-da333d00-3dfe-11ea-88cb-7a1ffef14241.png">
<img width="645" alt="Screen Shot 2020-01-14 at 12 40 56 PM" src="https://user-images.githubusercontent.com/12522275/73035983-da333d00-3dfe-11ea-892e-0653ceb3dcb1.png">

**Comment by MikeWalrath** [comment]
Closing. With the updated design I dont believe this issue to be relevant anymore.
